umpost/um2netcdf.py

```diff
@@ -635,6 +601,55 @@ def add_global_history(infile, iris_out):
     warnings.warn("um2nc version number not specified!")
 
 
+def rename_cube_var_name(c, um_var, simple: bool):
+    stash_code = c.attributes[STASH]
+
+    if simple:
+        # TODO: update formatting with fstrings
+        c.var_name = 'fld_s%2.2di%3.3d' % (stash_code.section, stash_code.item)
+    elif um_var.uniquename:
+        c.var_name = um_var.uniquename
+
+    # Could there be cases with both max and min?
+    if c.var_name:
+        if any([m.method == 'maximum' for m in c.cell_methods]):
+            c.var_name += "_max"
+        if any([m.method == 'minimum' for m in c.cell_methods]):
+            c.var_name += "_min"
+
+
+def rename_cube_names(c, umvar, verbose: bool):
```

Comment:
I'm not sure which would be cleaner here, but what would your opinion be on splitting this into two functions – one which handles the `standard_name`, and one which handles the `long_name`? E.g so we'd end up with a total of three renaming functions: `rename_cube_var_name`, `rename_cube_standard_name`, and `rename_cube_long_name` each handling a seperate part of the cube. It would require changing the tests though.

Reply:
Splitting these is a good idea, I didn't notice there's 2 separate blocks after reordering the naming logic.

Reply:
This is fixed in https://github.com/ACCESS-NRI/um2nc-standalone/pull/52/commits/763dedd03268b25e47475def88a8d08a2a0c12f2.